### PR TITLE
Suggest adding how to open migrated notebook

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3116

### CODE REVIEW

**Comment by get-itips** [comment]
@scanum

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/migrate-onenote-spmt.md [hunk 1]
```diff
@@ -41,3 +41,5 @@ When you open the OneNote application, they appear like this:
 Folders are migrated to SharePoint as **OneNote Notebook** content rather than a normal folder with files. They'll appear in SharePoint like this:
 
 ![OneNote migration setting6](media/onenote-file-5.png)
+
+For this **OneNote Notebook** to appear in your Notebooks list in OneNote, you can click the **Show Actions** ellipses, next to the notebook name in the Document library, then select **Open > Open in app**, remember first, to close in the OneNote application the already migrated notebooks.
```
Comment (JohanFreelancer9) [inline migration/migrate-onenote-spmt.md:45]:
```suggestion
For this **OneNote Notebook** to appear in your Notebooks list in OneNote, select the **Show Actions** ellipses next to the notebook name in the Document library, then select **Open > Open in app**. Remember to first close the OneNote application of the already migrated notebooks.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions.

**Review by scanum — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@tiburd Can you help me getting this merged? Thank you

**Pull request merged**